site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -42,28 +42,28 @@ include::docs/v1/tech/shared/_login_events.adoc[]
 
 == User Search
 
-User search requests may be made through the link:/docs/v1/tech/apis/users#search-for-users[User Search API] or within the FusionAuth admin UI under [breadcrumb]#Users#.
+[NOTE.since]
+====
 
 As of version 1.16.0, FusionAuth ships with a database search engine as the default.
 
-=== Configuration
+By selecting the appropriate link:/docs/v1/tech/installation-guide/fast-path[Fast Path installation], one can easily download a configuration with Elasticsearch pre-enabled.
 
-See the [field]#fusionauth-app.search-engine-type# and [field]#fusionauth-app.search-servers# properties, as well as the `FUSIONAUTH_SEARCH_ENGINE_TYPE` and `FUSIONAUTH_SEARCH_SERVERS` environment variables definitions of the link:/docs/v1/tech/reference/configuration/[Configuration Reference] for details on configuring the search engine and (optional) Elasticsearch integration.
+You can read more in our link:/docs/v1/tech/core-concepts/search[search core concept]
```

Comment:
Would change to -> "You can read more about the database and other search engine in the link:...[search core concepts section]."